angular/core/components/group_page/members_card/members_card.haml

```diff
@@ -3,7 +3,7 @@
   .members-card__list
     .members-card__avatar{ng-if: "group.memberships().length > 1", ng-repeat: "membership in group.memberships() | orderBy: '-admin' | limitTo:10"}
       %a.members-card__avatar-link{lmo-href-for: "membership.user()"}
-        %user_avatar{user: "membership.user()", coordinator: "membership.admin", size: "medium"}
+        %user_avatar{user: "membership.user()", coordinator: "membership.admin", size: "medium", alt: "{{membership.user().username}}"}
```

Comment:
Do you think this should be user().name rather than username? It will surely come across better in a screenreader..

Reply:
True, it might be better (although I'd be curious hearing the results in both cases!). And if the images can't load, it might be more legible too!